### PR TITLE
Add runtime.yml file

### PR DESCRIPTION
**Pull request by ganeshrn**
Depends-On: https://github.com/ansible/ansible/pull/67684

##### SUMMARY

*  Add minimum ansible base requirement (require_ansible)
*  Add action plugin redirection for modules (eg. junos_acls)
*  Add action plugin redirection for short module names (eg. acls)
*  Add short name module redirection (eg. acls -> junos_acls)
*  Add depcrecation notice for deprecated modules

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

meta/runtime.yml
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by pabelanger** [comment]
Until we land job redesign, I am going to close this issue (and reopen) to free up some capacity.

**Comment by pabelanger** [comment]
recheck

**Comment by danielmellado** [comment]
I've opened a PR for that, but noticed that this PR needs to be rebased first, please rebase, change that and recheck

**Comment by danielmellado** [comment]
FYI I created a rebased version here, https://github.com/ansible-collections/junipernetworks.junos/pull/57 as I couldn't do that over Ganesh's PR directly

File: meta/runtime.yml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+          and will be removed in a release after 2022-6-01. See the plugin documentation
+          for more details
+    lldp_interfaces:
+      redirect: junipernetworks.junos.junos_lldp_interfaces
+    logging:
+      redirect: junipernetworks.junos.junos_logging
+    netconf:
+      redirect: junipernetworks.junos.junos_netconf
+    ospf:
+      redirect: junipernetworks.junos.junos_ospf
```
[the remark this answers is not in the document]
Comment (ganeshrn) [reply meta/runtime.yml:240]: The module in the master branch is https://github.com/ansible-collections/junipernetworks.junos/blob/master/plugins/modules/junos_ospf.py that's the reason it redirects to junos_opsf

File: meta/runtime.yml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      redirect: junipernetworks.junos.junos
+    logging:
+      redirect: junipernetworks.junos.junos
+    junos_netconf:
+      redirect: junipernetworks.junos.junos
+    netconf:
+      redirect: junipernetworks.junos.junos
+    junos_ospf:
+      redirect: junipernetworks.junos.junos
+    ospf:
```
[the remark this answers is not in the document]
Comment (ganeshrn) [reply meta/runtime.yml:95]: Same as above

File: meta/runtime.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ganeshrn) [reply meta/runtime.yml:240]: I will update the PR after the PR to change junos_opsf -> junos_ospfv2 is merged as the runtime.yml file autogenerated based on modules in master by running the script https://github.com/ansible-network/collection_prep/blame/master/runtime.py

File: plugins/modules/junos_l2_interface.py [hunk 3]
```diff
@@ -13,7 +13,6 @@
 ---
 module: junos_l2_interface
 version_added: "1.0.0"
-module: junos_l2_interface
 author: Ganesh Nalawade (@ganeshrn)
 short_description: (deprecated) Manage Layer-2 interface on Juniper JUNOS network
   devices
```
Comment (rohitthakur2590) [inline plugins/modules/junos_l2_interface.py:18]: So I think this needs to be updated with script to have removed after as well.

**Pull request merged**